sequencer/src/MulticastReceiver.java

```diff
@@ -0,0 +1,31 @@
+import networkEntities.EntityAddressBook;
+
+import java.io.IOException;
+import java.net.DatagramPacket;
+import java.net.InetAddress;
+import java.net.MulticastSocket;
+import java.net.UnknownHostException;
+
+public class MulticastReceiver {
+   // public void receiveMulticastRequest(){
+   public static void main(String[] args) {
+        try{
+            InetAddress group = EntityAddressBook.REPLICA2.getAddress();
```

Comment:
the sequencer is not part of the group, so its multicast socket should not be joining